packages/builder/src/stores/portal/apps.js

```diff
@@ -65,6 +65,27 @@ export function createAppStore() {
     }
   }
 
+  async function updateIcon(appId, icon) {
```

Comment:
We could probably combine this with the `update` below so that we have a single function for logic around app updates, the implementations look pretty similar. 

Just a question, does the `put` endpoint act as a `patch`? Seems like it's fine if we send partial data by looking at usages of the two functions

Reply:
Yea it does, updated the function to handle both.